includes/rest/class-followers-controller.php

```diff
@@ -75,6 +75,52 @@ public function register_routes() {
 				'schema' => array( $this, 'get_item_schema' ),
 			)
 		);
+
+		// FEP-8fcf: Partial followers collection for synchronization.
+		\register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/followers/sync',
+			array(
+				'args' => array(
+					'user_id' => array(
+						'description'       => 'The ID of the actor.',
+						'type'              => 'integer',
+						'required'          => true,
+						'validate_callback' => array( $this, 'validate_user_id' ),
+					),
+				),
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_partial_followers' ),
+					'permission_callback' => array( 'Activitypub\Rest\Server', 'verify_signature' ),
+					'args'                => array(
+						'authority' => array(
+							'description' => 'The URI authority to filter followers by.',
+							'type'        => 'string',
```

Comment:
Is this a URI? Or could it be just a domain?

Reply:
It is the domain part but with the protocol.

The authority of `https://example.org/test?abc` is `https://example.org` so maybe we could call it `host` instead!?

Reply:
```suggestion
							'type'        => 'string',
							'format'      => 'uri',
							'pattern'     => '^https?://[^/]+$',
```
With this we should be able to skip the `// Validate authority format.` part below. Just the pattern would be enough for that actually. The uri format just sends it through `sanitize_url()`.